project/LibDependencies.scala

```diff
@@ -34,7 +34,7 @@ object LibDependencies {
     },
     play25 = {
       val compile = Seq(
-        "uk.gov.hmrc" %% "play-frontend-govuk" % "0.47.0-play-25"
+        "uk.gov.hmrc" %% "play-frontend-govuk" % "0.49.0-play-25"
```

Comment:
Shall we pull the version out into a string, like we do with hmrcFrontendVersion?